### PR TITLE
Ida Aspen Happy Thought

### PR DESCRIPTION
**Pull request by IdaAspen**
Link to my deployed project: https://aspen-happy-thought.netlify.app/

My reflections on how this week's project turned out: 
The challenge was to understand the components lifecycle and I needed to redo everything to be able to get it.

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/Components/GetThoughts.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const GetThought = ({ thought, onLikesIncrease }) => {
+  return (
+    <div className="get-thoughts__wrapper">
+      <p>{thought.message}</p>
+      <div className="get-thoughts__button-wrapper">
+        <button
+          type="submit"
+          className="get-thoughts__heart-button"
+          onClick={() => onLikesIncrease(thought._id)}
+          style={{ background: thought.hearts >= 1 ? '#ffadad' : '#eaeaea' }} // different colors on heart button if it has likes/no likes
```
Comment (brunsant) [inline code/src/Components/GetThoughts.js:15]: Nice inline styling!

File: code/src/Components/GetThoughts.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import './getThoughts.css';
+
+// Component for getting all thoughts
+const GetThought = ({ thought, onLikesIncrease }) => {
+  return (
+    <div className="get-thoughts__wrapper">
+      <p>{thought.message}</p>
+      <div className="get-thoughts__button-wrapper">
+        <button
+          type="submit"
```
Comment (brunsant) [inline code/src/Components/GetThoughts.js:12]: Is the type submit necessary here?

File: code/src/Components/InputCounter.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <p className="p-input-counter">
+      <span className={charCount > 140 ? 'p-red' : 'p-green'}>
+        {maxChar - charCount}
+      </span>
+      /{maxChar}
+    </p>
+  );
+};
+
+export default InputCounter;
```
Comment (brunsant) [inline code/src/Components/InputCounter.js:18]: Very nice component!! I will definitely try to improve my project with something like this later.

File: code/src/Components/header.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  display: flex;
+  justify-content: center;
+  padding: 0 20%;
+}
+
+header h1 {
+  font-size: 3.2em;
+  color: #ffff;
+  margin-bottom: 5%;
+  text-shadow: 5px 5px black;
```
Comment (brunsant) [inline code/src/Components/header.css:12]: The effect looks really great!! I initially thought you used a different font that was that way.

File: code/src/Components/newThoughtInput.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.new-thought-input__button:hover {
+  cursor: pointer;
+  border: 1px solid gray;
+}
+
+.new-thought-input__button:disabled {
+  opacity: 0.6;
+  cursor: not-allowed;
```
Comment (brunsant) [inline code/src/Components/newThoughtInput.css:48]: This is awesome! Such a simple detail that makes totally difference. I will definitely use this in the future.

**Review by brunsant — APPROVED** [review]
Amazing job Ida! 
The layout looks great and the code super clean and easy to follow!
The only comment I have is that it seems not all the thoughts are getting wrapped inside the box when they are too long (have no idea why some wrap and some don't). I had a similar situation and add   "overflow-wrap: break-word;" in the wrapper. Hope this can help.
Other than that, the page passed all validator tests and looks awesome in all browsers. 

Great job this week!!!

File: code/public/index.html [hunk 6]
```diff
@@ -4,31 +4,23 @@
...
-      Only files inside the `public` folder can be referenced from the HTML.
+    <meta property="og:title" content="Technigo Happy Thought" />
+    <meta property="og:description"
+      content="A happier version of twitter built in React. Collect Technigo students 'happy thoughts' using Fetch API." />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://aspen-happy-thought.netlify.app/" />
+    <meta property="og:image" content="https://aspen-happy-thought.netlify.app/images/happy-thought-og.jpg" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Oxygen+Mono&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (IdaAspen) [reply code/public/index.html:15]: Thanks Bruna, I pretty much have a default template that I use when setting up new projects (so you don't forget anything)

File: code/src/Components/GetThoughts.js (hunk 2 above)
Comment (IdaAspen) [reply code/src/Components/GetThoughts.js:12]: I did get a warning from ESlint, so doing this helped

**Pull request closed without merging**